packages/sw-background-sync-queue/src/lib/queue.js

```diff
@@ -6,16 +6,42 @@ import bcmanager from './broadcastManager';
 let _queue;
 let _config;
 
+/**
+ * Core queue class that handles all the enqueue and dequeue
+ * as well as cleanup code for the background sync queue
+ * @class Queue
+ */
 class Queue {
+	/**
+	 * Creates an instance of Queue.
+	 *
+	 * @memberOf Queue
+	 */
 	constructor() {
 		this.initialize();
 	}
 
+	/**
+	 * initialize the queue object from the idb
+	 * giving access to any penging queues
```

Comment:
I would drop "the" and just say "idb". I think there's a typo here for penging -> pending